src/utils/utils.jl

```diff
@@ -384,7 +384,7 @@ function copy_file(src::AbstractString, dst::AbstractString)
         if Sys.iswindows()
             run(`cmd /c copy /Y $(src) $(dst)`)
         else
-            return e
+            rethrow(e)
```

Comment:
This is technically correct but not the recommended way.  Just `rethrow()`.
```
  rethrow()

  Rethrow the current exception from within a catch block. The rethrown exception will continue propagation as if it
  had not been caught.

  │ Note
  │
  │  The alternative form rethrow(e) allows you to associate an alternative exception object e with the current
  │  backtrace. However this misrepresents the program state at the time of the error so you're encouraged to
  │  instead throw a new exception using throw(e). In Julia 1.1 and above, using throw(e) will preserve the
  │  root cause exception on the stack, as described in catch_stack.
```

Reply:
removed via 4db7838